tests/scripts/deploy-validate-vault.sh

```diff
@@ -206,9 +206,27 @@ function set_up_vault_kubernetes_auth {
   # create the RBAC for this SA
   kubectl -n "$ROOK_NAMESPACE" create clusterrolebinding vault-tokenreview-binding --clusterrole=system:auth-delegator --serviceaccount="$ROOK_NAMESPACE":"$ROOK_VAULT_SA"
 
-  # get the service account common.yaml created earlier
-  VAULT_SA_SECRET_NAME=$(kubectl -n "$ROOK_NAMESPACE" get sa "$ROOK_VAULT_SA" -o jsonpath="{.secrets[*]['name']}")
+  kubectl apply -f - <<EOF
```

Comment:
Could you add a comment here about why it's necessary to create the secret? The message in the commit message looks good, it would just be helpful to have it in the code as well IMO.

Reply:
added